WorldMoodTracker/js/components/Footer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from '../../../css/bootstrap.min.css';
+import custom from '../../css/style.css';
+
+export default class Footer extends React.Component {
+    render() {
+        return (
+            <footer>
+                <div className={styles.container}>
+                    <div className={[styles.hiddenLg, styles.hiddenMd, styles.hiddenSm, styles.textCenter,
+                        custom.footerLink].join(' ')}>
+                        <a href="/">Home</a>
```

Comment:
It would take you to the hosting's root. When deployed on `apps.loklak.org`, it would point to `apps.loklak.org`.

It is called home because for the app, home would be going back to http://apps.loklak.org web page.